pyiceberg/catalog/hive.py

```diff
@@ -111,6 +122,15 @@
 HIVE2_COMPATIBLE = "hive.hive2-compatible"
 HIVE2_COMPATIBLE_DEFAULT = False
 
+LOCK_CHECK_MIN_WAIT_TIME = "lock_check_min_wait_time"
+LOCK_CHECK_MAX_WAIT_TIME = "lock_check_max_wait_time"
+LOCK_CHECK_RETRIES = "lock_check_retries"
+DEFAULT_LOCK_CHECK_MIN_WAIT_TIME = 2
+DEFAULT_LOCK_CHECK_MAX_WAIT_TIME = 30
+DEFAULT_LOCK_CHECK_RETRIES = 10
```

Comment:
Should we align these properties with Java:

<img width="718" alt="image" src="https://github.com/apache/iceberg-python/assets/1134248/700d7689-4b70-4848-960d-6bd0fe1d8810">
https://iceberg.apache.org/docs/nightly/configuration/#write-properties

Reply:
A good idea, I have updated these default values.